src/app/routes/getInitialData/utils/fetchData/index.js

```diff
@@ -1,11 +1,20 @@
 import 'isomorphic-fetch';
 import nodeLogger from '#lib/logger.node';
 import preprocess from '#lib/utilities/preprocessor';
+import onClient from '#lib/utilities/onClient';
+import getBaseUrl from '../getBaseUrl';
 
 const logger = nodeLogger(__filename);
 const upstreamStatusCodesToPropagate = [200, 404];
 
-const fetchData = async ({ url, preprocessorRules }) => {
+const ampRegex = /(.amp)/g;
+
+const fetchData = async ({ pathname, preprocessorRules }) => {
+  const baseUrl = onClient()
+    ? getBaseUrl(window.location.origin)
+    : process.env.SIMORGH_BASE_URL;
+  const url = `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
```

Comment:
Might be worth a comment on this line just explaining why are stripping the `.amp` suffix